modules/build/src/main/scala/scala/build/options/BuildOptions.scala

```diff
@@ -500,13 +612,20 @@ final case class BuildOptions(
     val (scalaVersion, scalaBinaryVersion) =
       value {
         scalaOptions.scalaVersion match {
-          case Some("3.nightly") => computeLatestScalaThreeNightlyVersions()
-          case Some("2.nightly") => computeLatestScalaTwoNightlyVersions()
+          case Some("3.nightly")    => computeLatestScalaThreeNightlyVersions()
+          case Some("2.nightly")    => computeLatestScalaTwoNightlyVersions()
+          case Some("2.13.nightly") => computeLatestScalaTwoNightlyVersions()
+          case Some("2.12.nightly") => computeLatestScalaTwoTwelveNightlyVersions()
           case Some(versionString) if isScala3Nightly(versionString) =>
             turnScala3NightlyVersionArgIntoVersion(versionString)
           case Some(versionString) if isScala2Nightly(versionString) =>
             turnScala2NightlyVersionArgToVersions(versionString)
-          case _ => turnScalaVersionArgToScalaVersions(
+          case Some(versionString) if "[a-zA-Z]".r.findAllIn(versionString).nonEmpty =>
```

Comment:
```suggestion
          case Some(versionString) if versionString.exists(_.isLetter) =>
```

Reply:
Thanks for this style tip! I would apply it, then!